### DISCUSSION TITLE
[HomeAssistant] Prevent arming when a zone is open

### DISCUSSION THREAD

**Discussion by EmilMaric** [ParadoxAlarmInterface/pai#477] [Q&A]
I'm using HomeAssistant to control my Paradox alarm panel (SP6000). Was wondering if there is a way to prevent the alarm from arming if one of the zones is open?

I believe the Insite Gold app had something like this, wherein it would either alert you before arming if one of your zones was open and offer to bypass the zone for you, or just prevent the arming from happening. I don't remember exactly which of these it was but it was useful in case you went to arm your alarm and forget to close one of the zones (ex. arming when one of your windows is open). Looking to basically replicate the latter if possible.

**Comment by yozik04** [comment]
If you are arming from HomeAssistant I am not sure. If you are arming from panel then it is programmable with BabyWare.

**Comment by EmilMaric** [comment]
Ah yeah, I'm trying to control it from HomeAssistant :(
